### DISCUSSION TITLE
Can't download NanoMQ for Windows

### DISCUSSION THREAD

**Discussion by henjoe** [nanomq/nanomq#1114] [Q&A] [answered]
Hi I am trying to download NanoMQ for windows but seems like there is an error of the download link provided by their websites.

![image](https://user-images.githubusercontent.com/49782655/227682146-3bce4b30-7cbd-491a-84ff-6b6af9c95af2.png)

The error:

![image](https://user-images.githubusercontent.com/49782655/227682160-b5cabec3-6ac7-4fff-8f31-0a5c4c7d381f.png)

Regards,
Henjoe

**Comment by JaylinYu** [comment]
Sorry about it. Our devops is having troubles to fix related CI #1106 
I will upload one manually now

**Comment by JaylinYu** [comment, marked as answer]
Plz download windows msi installer from here https://github.com/emqx/nanomq/releases/tag/0.16.5

**Reply by henjoe** [reply]
Thanks I already downloaded and working right now! 

Regards,
Henjoe
